apps/admin-web/src/common/Topbar/Topbar.tsx

```diff
@@ -0,0 +1,18 @@
+import Link from 'next/link'
+import styles from './style.module.css'
+
+export default function Topbar() {
+  return (
+    <ul className={styles.navbar}>
+      <li className={styles.navbarItem}>
```

Comment:
FYI, normally, we don't use [CSS module](https://nextjs.org/docs/basic-features/built-in-css-support) because it's hard to read and hard to do conditional styling. Please use [emotion](https://emotion.sh/docs/introduction), or other modern CSS library instead.

Reply:
But you don't need to change them in this PR. Just FYI.